src/pages/home/report/ReportActionsView.js

```diff
@@ -177,8 +177,8 @@ function ReportActionsView(props) {
      * displaying.
      */
     const loadOlderChats = () => {
-        // Only fetch more if we are not already fetching so that we don't initiate duplicate requests.
-        if (props.isLoadingOlderReportActions) {
+        // Only fetch more if we are neither already fetching (so that we don't initiate duplicate requests) nor offline.
+        if (props.network.isOffline || props.isLoadingOlderReportActions) {
```

Comment:
This caused a regression https://github.com/Expensify/App/issues/31367. Previously even if we were offline  `Report.getOlderActions` would be still called and sets `isLoadingOlderReportActions` to `true` which in turns show the skeleton view. Now with this early return we prevented the skeleton view from showing up in this case.